File: src/main/java/com/clearcont/clearcontapp/repository/ResponsavelRepository.java
package com.clearcont.clearcontapp.repository;

import com.clearcont.clearcontapp.model.Responsavel;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface ResponsavelRepository extends JpaRepository<Responsavel, Long> {
    List<Responsavel> findResponsavelByEmpresa_Id(Long id);
}

File: src/main/kotlin/br/com/clearcont/clearcontwebapp/repository/ResponsavelRepository.kt
package br.com.clearcont.clearcontwebapp.repository

import br.com.clearcont.clearcontwebapp.models.Responsavel
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ResponsavelRepository : JpaRepository<Responsavel?, Long?> {
    fun findResponsavelByEmpresa_Id(id: Long?): List<Responsavel?>?
}
